packages/maskbook/src/web3/hooks/stables_coins.json

```diff
@@ -0,0 +1,246 @@
+{
```

Comment:
Do not copy & paste the response. Make it an array of `ERC20TokenDetailed` entities.